test/controllers/locks_controller_test.rb

```diff
@@ -270,12 +270,12 @@ def create_lock(resource = nil, options = {})
       end
     end
 
-    describe "#destroy_via_resource" do
+    describe "#destroy via resource" do
```

Comment:
not sure I understand the "via resource" part here

Reply:
old name was kinda weird ... changed it to `destroy via query`

Reply:
... we don't really need this since api users could just hit index and find the lock they want to deleted ... old cruft :(